src/modules/fancyzones/editor/FancyZonesEditor/App.xaml.cs

```diff
@@ -13,12 +15,19 @@ namespace FancyZonesEditor
     /// </summary>
     public partial class App : Application
     {
-        public Settings[] ZoneSettings { get; }
+        public static Settings[] ZoneSettings { get; set; }

```

Comment:
I'm not sure if these really belong under App. You end up calling App.ZoneSettings[MonitorVM.CurrentMonitor] a lot of times. Maybe you want to make a property that returns that instead. E.g. you could move this array to the Settings class, make it private, and add a property like this

public static Settings Current{
 get{ return App.ZoneSettings[MonitorVM.CurrentMonitor],}
}

Then you only need to call Settings.Current everywhere

Reply:
That's really neat! I tried just changing all the fields and everything crashed buuut... I'll do my best :D